jetstream/config.py

```diff
@@ -71,7 +71,11 @@ def with_configs_from(
         config_collection = ConfigCollection.from_github_repos(
             repo_urls=repo_urls, is_private=is_private
         )
-        self.configs.merge(config_collection)
```

Comment:
When using `with_configs_from` on a fresh `ConfigLoader` we don't want to pull in metric-hub and merge it into the custom paths since this might overwrite specific configs leading to faulty validation. However by calling `self.configs` we always pull in configs from metric-hub `main`.